### PR TITLE
Support release note local preview

### PR DESCRIPTION
**Pull request by LitoMore**
### Changes

- Add support for previewing the release note on the developer's local
- Comment the data file output out because it's quite long
- Bump latest Jest

Use the command below:

```shell
SI_REPOSITORY_TOKEN='your-repo-token' npm run preview
```

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -24,6 +19,14 @@ Please note that modifications should follow these coding guidelines:
 
 ## Building
 
-We use [@zeit/ncc](https://github.com/zeit/ncc#readme) to create a single `.js` file that contains all the source code needed to run the Action. This means that this file works without the `node_modules/` folder present.
+We use [@zeit/ncc](https://github.com/vercel/ncc#readme) to create a single `.js` file that contains all the source code needed to run the Action. This means that this file works without the `node_modules/` folder present.
```
Comment (adamrusted) [inline CONTRIBUTING.md:22]:
```suggestion
We use [@vercel/ncc](https://github.com/vercel/ncc#readme) to create a single `.js` file that contains all the source code needed to run the Action. This means that this file works without the `node_modules/` folder present.
```

**Review by mondeja — APPROVED** [review]
Great!

File: src/create.js [hunk 2]
```diff
@@ -308,9 +308,6 @@ async function getChangesFromFile(core, file, client, context, id) {
     core.info(`Detected a change to the data file`);
     const changes = [];
 
-    core.debug(`\nSimple Icons data file`);
-    core.debug(file);
```
Comment (mondeja) [inline src/create.js:312]: I think that this is still useful.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -24,6 +19,14 @@ Please note that modifications should follow these coding guidelines:
...
+We use [@vercel/ncc](https://github.com/vercel/ncc#readme) to create a single `.js` file that contains all the source code needed to run the Action. This means that this file works without the `node_modules/` folder present.
 
 Therefore, you should make sure the `npm run build` command does not fail due to your changes. Run `npm run build` before making a commit and include the changes in the `./lib` folder in your commit.
+
+## Local Preview
+
+Use this command below to preview the release note:
+
+```shell
+SI_SI_REPOSITORY_TOKEN='your-repo-token' npm run preview
```
Comment (mondeja) [inline CONTRIBUTING.md:31]:
```suggestion
SI_REPOSITORY_TOKEN='your-repo-token' npm run preview
```

File: package.json [hunk 4]
```diff
@@ -11,6 +11,7 @@
     "email": "ericornelissen@gmail.com"
   },
   "scripts": {
+    "preview": "GITHUB_REPOSITORY=simple-icons/simple-icons node src/index.js",
```
Comment (mondeja) [inline package.json:14]: This environment variable `GITHUB_REPOSITORY`is used?

Comment (LitoMore) [reply package.json:14]: Yes, it's used by the octokit. You can try without this variable to see the error.

**Review by mondeja — APPROVED** [review]

**Pull request merged**